main.go

```diff
@@ -216,14 +206,12 @@ func TcpReceiver(done <-chan struct{}, af string, targetAddr string, probePortSt
 	return out, nil
 }
 
-//
-// This runs on its own collecting Icmp responses until its explicitly told to stop
-//
-func IcmpReceiver(done <-chan struct{}, af string) (chan interface{}, error) {
+// ICMPReceiver runs on its own collecting Icmp responses until its explicitly told to stop
```

Comment:
I don't get what you mean.

To clarify: I'm asking if you can amend your diff to make the word "Icmp" uppercase in that comment, for consistency with the rest